packages/components/src/index.js

```diff
@@ -96,3 +96,4 @@ export { default as withNotices } from './higher-order/with-notices';
 export {
 	default as withSpokenMessages,
 } from './higher-order/with-spoken-messages';
+export * from './text';
```

Comment:
Hmm 🤔 . This will be the first time the current entry `index.js` will export an Emotion styled component (and mixins) in this fashion. Because of that, it makes sense that it's a `*` from the `./text` file.

Right now, it looks like the bottom chunk of the file has HOC exports grouped, with regular components at the top.

What do you think about moving this export to the top group (under `TabPanel` for alphabetized listing)

Reply:
I think it's too soon to export any mixins... We should use them internally in the package but just export components outside.

Reply:
I've removed the export and documentation for the mixin.